packages/commons/src/file-manager/fileManager.ts

```diff
@@ -44,6 +44,12 @@ export type FileManager = {
     },
     logger: Logger
   ) => Promise<string>;
+  storeBytesByPath: (
+    bucket: string,
+    path: string,
```

Comment:
We should use the one implemented by @Viktor-K https://github.com/pagopa/interop-be-monorepo/pull/973/files#diff-324984850cb2cfa0887fc0712332903b6480dadacd8ffb0b1869107bf67444afR47

Reply:
Perfect, i will just cherry pick it from the given PR :)